### DISCUSSION TITLE
Difference in Card border-width on storybook vs actuals

### DISCUSSION THREAD

**Discussion by falconAura** [twilio-labs/paste#2993] [General] [answered]
Hi Team,
I am using the Card component on my webapp and noticed that it has a border-width set to 2px. Reference: https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/card/src/index.tsx#L24

However, when I check the storybook, the border-width calculated in safari and chrome is **1px**. 

Is this a bug?  If so, can you please fix it? 

<img width="444" alt="Screenshot 2023-01-30 at 1 23 39 AM" src="https://user-images.githubusercontent.com/117545898/215440222-cf281b28-2693-4842-8796-6fb9eab200d7.png">

**Comment by gloriliale** [comment, marked as answer]
Hi @falconAura! The latest version of Card ([6.0.3](https://paste.twilio.design/components/card#6-0-3)) released last Jan 25 in core v 17.5.0 includes a style update that affected the width of this component's border, as well as its border radius and padding. If you update to this new version, the Card in your webapp will have this updated 1px border. Let us know if you have any more questions!
